lib/samson/secrets/vault_client_wrapper.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'vault'
+
+module Samson
+  module Secrets
+    class VaultClientWrapper < Vault::Client
+      # TODO: make these configurable via env vars and send env vars to samson_secret_puller
+      MOUNT = 'secret'
+      PREFIX = 'apps'
```

Comment:
I'd rather have the manager own the mount + prefix ... this thing here is more of a crutch and will be deleted eventually